app/models/block.rb

```diff
@@ -8,10 +8,15 @@ class Block < ActiveRecord::Base
   validates :person_id, :presence => true, :uniqueness => { :scope => :user_id }
 
   validate :not_blocking_yourself
+  validate :not_blocking_podmin
 
   def not_blocking_yourself
     if self.user.person.id == self.person_id
       errors[:person_id] << "stop blocking yourself!"
     end
   end
+
+  def not_blocking_podmin
+    errors[:person_id] << t('blocks.create.failure.podmin') if Role.exists?(person_id: self.person_id, name: "admin")
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Redundant `self` detected.